README.md

```diff
@@ -74,8 +75,40 @@ const UserController = () => {
 export default UserController; // module.exports = UserController;
 ```
 
+## Middlewares
+Middlewares allow you perform any set of operation on a particular route. They are executed from **top-to-bottom**, as they are arranged in the `middlewares` array.
 
-### Express with mapped Routes
+To proceed to the next middleware or the controller, never forget to call the `next()` function.
+
+For more examples, See [Middleware Example](./examples/app/config/routes.js).
+
+```Javascript
+const checkIfAutheticated = (req, res, next) => {
+  console.log('authenticated');
+  next();
+};
+
+const verifyFacebookAuth = (req, res, next) => {
+  console.log('unverified');
+  return res
+    .status(400)
+    .json({status: false, message: 'Sorry, you aren\'t authorized on facebook'});
+};
+
+const routes = {
+  'GET /user:id': {
+    path: 'UserController.get',
+    middlewares: [
+         checkIfAutheticated,
+         verifyFacebookAuth,
+    ],
+    
+  'POST /user': 'UserController.create'
+  },
```

Comment:
This `}` is on the wrong location... `'POST /user': 'UserController.create'` should be outside of the object...